doc/changes/changes_1.2.0.md

```diff
@@ -0,0 +1,32 @@
+# Exasol Row Level Security (Lua) 1.2.0, released 2022-05-??
+
+Code name: `ORDER BY`, `LIMIT`, more functions and unified error handling
+
+## Summary
+
+Release 1.2.0 of `row-level-security-lua` upgrades the base module to `virtual-schema-common-lua` 1.1.0, which brings support for the `ORDER BY` and `LIMIT` clause as well as more scalar function coverage and uniform error reporting. 
+
+Note that this version requires Exasol 7.1.10 or later to run stable, because earlier versions had a problem in handling Lua's `pcall` and `xpcall` functions. Please upgrade your database before running RLS!
+
+We also added validation for Virtual Schema properties to all requests that use properties.
+
+### Know Issues
+
+Push-down of aggregate functions is not yet implemented. We will add that in a future version.
```

Comment:
If available please add a link to the ticket where this will be implemented.

Reply:
Done (https://github.com/exasol/row-level-security-lua/issues/120).